CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ release.
 
 ## Unreleased
 
+- Remove `network.protocol.name` from HTTP semconv.
+  ([#398](https://github.com/open-telemetry/semantic-conventions/pull/398))
```

Comment:
Can this be represented in schema-next.yml as well?

Reply:
not that I'm aware of